Lib/test/dh2048.pem

```diff
@@ -0,0 +1,29 @@
+    DH Parameters: (2048 bit)
+        prime:
+            00:de:18:82:8e:24:af:04:cf:9d:32:d2:88:56:8a:
+            14:fa:52:53:16:dc:f6:38:86:2b:99:54:ae:1e:84:
+            86:e5:77:23:3a:6d:5b:c3:b8:cb:32:71:d0:73:17:
+            c0:e8:c6:ad:ee:bf:70:df:ae:d1:00:99:c7:23:79:
+            f9:47:2c:29:1c:a7:8c:5b:36:1a:e6:70:54:4a:67:
+            3c:80:0a:0b:d3:4d:36:1e:34:c8:a0:1a:8f:8a:a1:
+            a7:fc:cd:e0:db:80:f9:38:c0:56:25:71:c4:87:25:
+            fc:ab:17:37:5e:02:eb:01:16:03:ae:10:0f:cc:94:
+            d7:62:26:00:7f:86:4d:9a:7e:1c:06:ef:a5:04:eb:
+            8f:86:23:dd:d7:ca:ee:e7:e8:36:12:57:50:14:db:
+            31:99:a2:26:8c:43:db:7f:01:18:33:b9:97:75:09:
+            ea:b9:0e:57:6c:31:b4:11:b8:6b:f7:7d:d6:ad:6f:
+            3f:ff:16:32:51:a7:6d:f3:0f:5b:b0:16:e9:e8:00:
+            8d:73:60:ae:3b:23:e5:82:bf:2e:ef:fd:b3:2d:6f:
+            97:9c:f9:69:24:38:65:9e:29:39:87:ba:e7:93:d0:
+            82:1e:06:47:02:7f:ac:fc:8b:5f:1a:14:c3:c0:ee:
+            28:81:d1:48:7d:78:f0:17:dc:c8:aa:ae:07:b7:39:
+            56:cb
+        generator: 2 (0x2)
```

Comment:
one improvement towards future-compatibility could be to use standardized DH prime parameters e.g., from [rfc7919](https://tools.ietf.org/html/rfc7919). Thinking of future implementation change which could verify that the primes used come from a known set params.